client/branded/src/components/panel/Panel.tsx

```diff
@@ -221,19 +223,31 @@ export const Panel = React.memo<Props>(props => {
         setTabIndex(items.findIndex(({ id }) => id === currentTabID))
     }, [items, hash, currentTabID])
 
-    const activeTab: PanelItem | undefined = items[tabIndex]
+    const [isRedesignEnabled] = useRedesignToggle()
+    const EmptyPanelWrapper = isRedesignEnabled ? React.Fragment : 'div'
 
     if (!areExtensionsReady) {
-        return <ExtensionsLoadingPanelView />
+        return (
+            <EmptyPanelWrapper {...(isRedesignEnabled && { className: 'panel' })}>
```

Comment:
```suggestion
            <EmptyPanelWrapper {...(!isRedesignEnabled && { className: 'panel' })}>
```

Should this logic not be the other way around? Otherwise you are applying `className` to `React.Fragment`

Reply:
Yeah, I didn't push my last commit 🙂